assets/src/js/admin/plugin-deactivation-survey.js

```diff
@@ -56,9 +56,9 @@ class GiveDeactivationSurvey {
 
 					// If no radio button is selected then throw error.
 					if ( 0 === checkedRadio.length && 0 === deactivationError.length ) {
-						surveyForm[ 0 ].innerHTML += `
+						surveyForm[0].innerHTML += `
 							<div class="notice notice-error deactivation-error">
-								${ Give.fn.getGlobalVar('deactivation_no_option_selected') }
+								${ Give.fn.getGlobalVar( 'deactivation_no_option_selected' ) }
```

Comment:
@raftaar1191 did you forget to edit this?

Reply:
Yup @ravinderk I remove it now